### PR TITLE
UVIS fixes and unit tests

### PR DESCRIPTION
**Pull request by rosteen**
Draft for now, more work to do on the unit tests including rebasing to include the updates from the G102 PR.

### CODE REVIEW

**Comment by rosteen** [comment]
@duytnguyendtn Rebased, ready for final review.

**Review by duytnguyendtn — COMMENTED** [review]
Since we're merging as is, I think this is good coverage for the time being. One small difference I noticed in our definition of "roundtripping" that, though minor, is something I feel should be highlighted, as I imagine we'll be migrating this hardcoded uvis tests to the parameterized tests I developed:

In this PR, it looks like "round tripping" is being done by comparing the results to the expected, then sending the expected back through and comparing to the original. In the existing WFC3 IR tests, I implemented roundtripping as calculating the original, comparing to the expected, THEN sending the original back through and comparing the results to the original input. Not sure if the distinction is important, but it's one I wanted to capture, and hopefully I explained it well enough...?

As another smaller point, I noticed that the roundtripping is only for the grism to world distortion, not the detector to grism distortion. Though I vaguely remember there being a complication with the latter inverse distortion, so if I could be reminded of that, that would be great! 

Regardless, these changes are minor, and I'll go ahead and approve immediately after the small cleanup suggestion is addressed

File: astrogrism/tests/test_wfc3_g280.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
+from astrogrism import GrismObs
+
+import pathlib
+import numpy as np
+
+# TODO: Switch to importlib
+test_dir = pathlib.Path(__file__).parent.absolute()
+grism_image_file = 'https://github.com/npirzkal/aXe_WFC3_Cookbook/raw/main/cookbook_data/G141/ib6o23rsq_flt.fits' # noqa
```
[the remark this answers is not in the document]
Comment (rosteen) [reply astrogrism/tests/test_wfc3_g280.py:8]: Whoops, good call. Missed deleting that when I copied another old test file to start from.

**Comment by duytnguyendtn** [comment]
Ahh I missed that. Thanks!

**Pull request merged**